frontend/src/features/user/mutations.ts

```diff
@@ -50,9 +53,26 @@ export const useUserMutations = () => {
     },
   })
 
+  const transferOwnershipMutation = useMutation(
+    (params: TransferOwnershipRequestDto) => transferOwnership(params),
+    {
+      onSuccess: (_) => {
+        toast({
+          description: 'Ownership transferred.',
+        })
+      },
+      onError: (error: ApiError) => {
+        failureToast({
```

Comment:
@tshuli there's already a failure toast.

After seeing your [review comment](https://github.com/opengovsg/FormSG/pull/6087#discussion_r1278807576), I thought of moving this toast call out but I see that this file already has toast calls and thought it's a good idea to centralise the calls.

Lmk wdyt?

Reply:
Ah okay, i think the current implementation is ok then! Thanks for highlighting